tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -196,8 +196,31 @@ def test_brand_string(test_microvm_with_api, network_config):
 ]
 # fmt: on
 
-# CPU templates which support MSR test
-MSR_SUPPORTED_TEMPLATES = ["T2A", "T2S"]
+
+def get_msr_supported_templates():
+    """
+    Return the list of CPU templates supported for MSR-related tests.
+    """
+    # CPU templates supported for the MSR tests
+    _msr_supported_templates = ["T2A", "T2S"]
```

Comment:
nit: any reason it starts from underscore?

Reply:
underscore was added to highlight that this is a private variable. If it isn't inline with our coding conventions I can remove it.

Reply:
No, I'm not aware of such a convention. At least by looking at this file I can't see other examples like that.
The only similar thing that come to my mind is __test_*() functions that are prepended with an underscore to hide them from pytest.